test/fixtures/bootstrap/main.tf

```diff
@@ -22,4 +22,5 @@ module "bootstrap" {
   group_billing_admins = var.group_email
   billing_account      = var.billing_account
   org_project_creators = var.org_project_creators
+  project_prefix       = "tst"
```

Comment:
I think that to help maintainability, instead of using

```
project_prefix       = "tst"
```

here and in the other fixture files we cloud use 

```
project_prefix       = var.project_prefix
```

and set  `tst` only in the cloud build file

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/build/int.cloudbuild.yaml

what do you think @bharathkkb